src/lib/adapters/waku/index.ts

```diff
@@ -453,4 +458,22 @@ export default class WakuAdapter implements Adapter {
 		}
 		balanceStore.set(balancesState)
 	}
```

Comment:
Do we still need this?

Reply:
Yeah, because this is at the intersection of two issues:
https://github.com/logos-innovation-lab/waku-objects-playground/issues/168 and https://github.com/logos-innovation-lab/waku-objects-playground/issues/171

First of all we need to initialize the balances somewhere when an account is created. Currently there is no real support for multiple currencies (at least not with the transactions) but the UI supports them and removing this code would only show the native token.

Secondly, with the Waku adapter the balance is not stored on the network, so it has to come from somewhere. In the case of the native token it comes from the blockchain, and for other currencies it is hardcoded. I think we should first fix the support for non-native tokens and then the Firebase should follow the same pattern. Then we can get rid of this function.

Reply:
I see. But for the waku adapter we can for now only support native token. That's completely fine, we don't need to have additional token. But I am also OK with keeping this code here.

Reply:
I prefer to keep the feature parity, because there are less surprises later.